scripts/CompareFitMinimizers/results_output.py

```diff
@@ -74,14 +80,23 @@ def print_group_results_tables(minimizers, results_per_test, problems_obj, group
         header += "\n\n"
         print(header)
         print (tbl_acc_indiv)
+        rel_col_defs = '../../docs/source/concepts/minimizers_comparison/color_definitions.txt'
+        color_definitions = os.path.join(SCRIPT_DIR, rel_col_defs)
 
-        # optionally save the above table to file
+        # optionally save the above table to a .txt file and a .html file
         if save_to_file:
```

Comment:
The number of lines within the save_to_file I would say now justify a new method called something like save_table_to_file